packages/scripts/src/commands/app-data/convert/index.ts

```diff
@@ -38,7 +38,7 @@ export default program
 
 class AppDataConverter {
   /** Change version to invalidate any cached conversions */
-  public converterVersion = 1;
+  public converterVersion = 1.1;
```

Comment:
internal code used simply to invalidate previously converted files and re-convert (to process with new legacy mapping update)